src/desktop/apps/authentication/index.ts

```diff
@@ -17,9 +18,11 @@ app.get("/signup", redirectLoggedInHome, signup)
 app.get("/forgot", forgotPassword)
 app.get("/reset_password", resetPassword)
 
-app.get("/log_in", (_req, res) => {
-  res.redirect(301, "/login")
+app.get("/log_in", (req, res) => {
+  const parsedUrl = parse(req.url)
+  res.redirect(301, `/login?${parsedUrl.query}`)
 })
-app.get("/sign_up", (_req, res) => {
-  res.redirect(301, "/signup")
+app.get("/sign_up", (req, res) => {
+  const parsedUrl = parse(req.url)
+  res.redirect(301, `/signup?${parsedUrl.query}`)
```

Comment:
Is there a better way to accomplish this?

Reply:
A better way to redirect? Or a better way to get query params? If the latter, should be able to do [`req.query.someParam`](https://coderrocketfuel.com/article/handle-get-request-query-string-parameters-in-express-js), but this also works fine since your translating it back into a string.

Reply:
I meant a better way to get the full querystring.